xml/depl_maintenance.xml

```diff
@@ -448,8 +448,10 @@
     </listitem>
     <listitem>
      <para>
-      Upgrading is not supported if &o_blockstore; has been deployed with the
-      <literal>raw devices</literal> back-end.
+       Note that a non-disruptive upgrade is not supported if the
+       &o_blockstore; has been deployed with the <literal>raw devices</literal>
+       or <literal>local file</literal> back-end. In this case, you have to
+       perform a regular upgrade.
```

Comment:
User either has to perform 'normal' upgrade, or there might be a chance to change the cinder backend (just like with other checks for possibility of non-disruptive upgrade)